target/linux/rockchip/image/armv8.mk

```
@@ -135,6 +135,7 @@ define Device/pine64_rockpro64
   DEVICE_VENDOR := Pine64
   DEVICE_MODEL := RockPro64
   SOC := rk3399
+  SUPPORTED_DEVICES := pine64,rockpro64 pine64,rockpro64-v2.1
```

This just adds the new name:
```suggestion
  SUPPORTED_DEVICES += pine64,rockpro64-v2.1
```

---

Cheers for the review - I've made the suggested change :) 

For anyone else that is curious https://github.com/openwrt/openwrt/blob/0ee1ede25a9f8278a8e0bc0820e4c55d2023adf9/include/image.mk#L514 is where I believe this value initially gets set.